pkgs/flox-pkgdb/default.nix

```diff
@@ -0,0 +1 @@
+{inputs, ...}: inputs.pkgdb.packages.flox-pkgdb
```

Comment:
I think we should capacitate this at some point so we don't get multiple versions of nixpkgs in our flox closure? But not necessary right now

Reply:
Because this is a dependency of `flox`, and capacitor depends on `floxpkgs`, and `floxpkgs` depends on `flox` - attempting to capacitate this project would introduce a dependency cycle.

We ran into issues with this previously attempting to capacitate `parser-util`.